scene/2d/parallax_layer.cpp

```diff
@@ -87,6 +87,9 @@ void ParallaxLayer::set_mirroring(const Size2 &p_mirroring) {
 	if (mirroring.y < 0)
 		mirroring.y = 0;
 
+	if (!is_inside_tree())
```

Comment:
Perhaps it would be better to check this directly in `_update_mirroring` method as to avoid the same bug happening in the future, even if this is called in `NOTIFICATION_ENTER_TREE` already.

Reply:
Ok, I'm moving the test inside `_update_mirroring()`.